src/index.js

```diff
@@ -125,10 +125,10 @@ export default {
     )
   },
 
-  del (path: string, options?: {} = {}): Request {
+  del (path: string, data?: {}, options?: {} = {}): Request {
```

Comment:
why `del` and not `delete`?

Reply:
https://www.youtube.com/watch?v=-2-40PcbK5Y

Reply:
I guess I saw rust going from `function` to `fn` and I thought... 🤔 hmm.. there is a trend here.